nixos/modules/services/system/nscd.nix

```diff
@@ -37,6 +37,10 @@ in
   ###### implementation
 
   config = mkIf cfg.enable {
+    system.activationScripts.netgroup = ''
+      touch /etc/netgroup
```

Comment:
`systemd.tmpfiles.rules` is preferred here. If we ever want to manage this file or a user would it not be better to use `environment.etc."netgroup"`?

Reply:
My understanding was it could be generated by some tools similar to how resolv.conf works. But unclear on that, and probably need someone who actually uses this file to provide some input. This was just the easiest way to silence that warning.

Reply:
Something like following should work too:

```
systemd.tmpfiles.rules = [ "f /etc/netgroup" ];
```